config/locales/en.yml

```diff
@@ -639,6 +639,44 @@ en:
             one: "A new job matching your search criteria '%{reference}' was posted yesterday."
             other: "%{count} new jobs matching your search criteria '%{reference}' were posted yesterday."
           unsubscribe: Don’t want to receive these email alerts? %{unsubscribe_link}
+  nqt_job_alerts:
+    page_title: Sign up for NQT job alerts
+    heading: Sign up for NQT job alerts
+    intro:
+      teaching_vacancies: >-
+        Teaching vacancies is a free job-listing service from the Department for Education. It helps schools to save
```

Comment:
Typo: 'vacancies' should be capitalised: 'Vacancies'.

Reply:
I'm just a dev; a mere mortal 😢 . I'll check with the content wizard @valentine-carter